var/spack/repos/builtin/packages/py-bluepyefe/package.py

```diff
@@ -0,0 +1,28 @@
+##############################################################################
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class PyBluepyefe(PythonPackage):
+    """ Blue Brain Python E-feature extraction"""
+
+    homepage = "https://github.com/BlueBrain/BluePyEfe"
+    url = "https://pypi.io/packages/source/b/bluepyefe/bluepyefe-0.3.13.tar.gz"
```

Comment:
```suggestion
    url = "https://pypi.io/packages/source/b/bluepyefe/bluepyefe-0.3.13.tar.gz"

```
Sorry, there should still be a newline.